FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Tool/Microsoft.Health.Fhir.Synapse.Tool.csproj

```diff
@@ -11,6 +11,7 @@
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="5.0.0" />
     <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="5.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
+    <PackageReference Include="Microsoft.Health.Fhir.Synapse.Parquet" Version="1.0.0.41" />
```

Comment:
Nit: Maybe we don't need to include the Parquet lib here.?

Reply:
We need the lib here to make `copy native libs` happen during nuget restore phase